style.css

```diff
@@ -1 +1,194 @@
 /*  Add your styling here */
+body{
+    margin: 0;
+    font-family: Arial, Helvetica, sans-serif;
+    display: grid;
+    grid-template-areas: 
+    "header"
+    "main"
+    "main2"
+    "footer";
+
+}
+a {
+    color: rgb(122, 0, 122);
+}
+
+
+.header1{
+    grid-area: header;
+    background-color: rgb(235, 195, 123);
+    position: fixed;
+    box-shadow: 1px 1px 4px rgba(0,0,0,.2);
+    width: 100%;
+    z-index: 10;
+    height:75px;
+}
+
+.header1 ul{
+    list-style: none; 
+    overflow: hidden;
+}
+
+.header1 ul a{
+    display: block;
+    /* outline: 1px solid purple; */
+    padding:15px;
+    border:1px solid purple;
+    text-decoration: none;
+}
+
+.header1 .logo{
+    float: left;
+    display: block;
+    margin-top: 5px;
+    margin-left: 5px;
+}
+.header1 ul a:hover{
+    background-color:rgb(226, 165, 52);
+    text-decoration: none;
+}
+.header1 .menu{
+    clear: both;
+    max-height: 0;
+    transition: max-height .2s ease-out;
+}
+.header1 .menu-icon{
+    /* border: 1px solid purple; */
+    padding: 28px 20px;
+    position: relative;
+    float: right;
+    cursor: pointer;
+}
+.header1 .menu-icon .nav-icon{
+    background: black;
+    display: block;
+    height: 2px;
+    width: 18px;
+    position: relative;
+    transition: background .2s ease-out;
+}
+.header1 .menu-icon .nav-icon:before{
+    background: black;
+    content:"";
+    display: block;
+    height: 100%;
+    width: 100%;
+    position: absolute;
+    transition: all .2s ease-out;
+    top: 5px;
+    
+}
+.header1 .menu-icon .nav-icon:after{
+    background: black;
+    content:"";
+    display: block;
+    height: 100%;
+    width: 100%;
+    position: absolute;
+    transition: all .2s ease-out;
+    top: -5px;
+}
+.header1 .menu-btn{
+    display: none;
+}
+.header1 .menu-btn:checked ~ .menu{
```

Comment:
You probably don't need the `1` after all of these names and it would easier to read if you didn't :)

Maybe try renaming them

Reply:
ok, I did it.